src/gui/macutilities.mm

```diff
@@ -35,18 +35,21 @@
 #include <QSet>
 #include <QSize>
 #include <QString>
-#include <QtMac>
+
+QImage qt_mac_toQImage(CGImageRef image);
```

Comment:
This function was used internally to implement `QtMac::fromCGImageRef()`. It's still in use in Qt6 and it's available as an imported function, so we'll be able to use it for now so we don't have to borrow all that code.